pyblish_ftrack/plugins/conform_ftrack_component_preview.py

```diff
@@ -0,0 +1,43 @@
+import pyblish.api
+import os
+import ftrack
+
+@pyblish.api.log
+class FtrackUploadPreview(pyblish.api.Conformer):
+    """Creates Preview component within supplied version, also makes this preview reviewable.
+
+    Expected data members:
+    'publishedFile' - path that will be saved as a component
+    'ftrackVersionID' - ID of a version where component should be created
```

Comment:
That would work nicely. I just put it there without much though for formatting. Maybe I'd change the name 'Arguments' to something else so it's not confused with actual arguments.